conf/runner.yaml

```diff
@@ -691,3 +691,8 @@ smtp:
     #auth:
     #    user: someuser
     #    pass: somepass
+
+slack:
+    username: tls-observatory
+    icon_emoji: ':telescope:'
```

Comment:
Is this means to be a slack keyword that displays a specific emoji in channel? If so, could you add a comment to explain it, and maybe put a link to a list of acceptable emojis?